pkg/apis/machineconfiguration.openshift.io/v1/helpers.go

```diff
@@ -23,10 +24,15 @@ func MergeMachineConfigs(configs []*MachineConfig, osImageURL string) *MachineCo
 	for idx := 1; idx < len(configs); idx++ {
 		outIgn = ign.Append(outIgn, configs[idx].Spec.Config)
 	}
+	kargs := []string{}
+	for _, cfg := range(configs) {
```

Comment:
you don't need parenthesis around `config` - also, why this loop here? `configs` and `cfg.Spec.KernelArguments` are slides so order is maintained anyway (and `generateKargsCommand` now loops over slices as well so order is maintained as well and we're still O(1) on checks).

Reply:
Dropped the parens.  The loop is to merge (concatenate) the kargs, just like we're merging together the Ignition specs right?

Reply:
> The loop is to merge (concatenate) the kargs, just like we're merging together the Ignition specs right?

damn I must be blind, sorry for the noise